### ISSUE TITLE
move MovementDescriber to scenery-phet

### ISSUE THREAD

**Issue by zepumph** [phetsims/scenery-phet#445]
I think we should do this now rather than later. I'd be curious to see if @mbarlow12 could utilize this in his magnet describer code in FL.

Once it is over in scenery-phet, we should factor out strings into that string js file.

**Comment by zepumph** [comment]
This issue has gotten really out of date. Unassigning until I work on Movement describer again.

**Comment by pixelzoom** [comment]
Reviewing scenery-phet issues for Q4 2022 planning...

@zepumph status of this issue? I see no MovementDescriber, maybe it's MovementAlerter now?   And it does live in scenery-phet.

**Comment by jessegreenberg** [comment]
I am not sure about the remaining work for this one, @zepumph can it be closed?

**Comment by zepumph** [comment]
Yes. The work outlined in https://github.com/phetsims/scenery-phet/issues/445#issuecomment-435210375 makes sense, but it hasn't been important or useful in the last 4 years of description implementation. We can hold off until we hit this again, or just make a new implementation then. Closing
